app/public/js/location_search_map.js

```diff
@@ -118,4 +120,27 @@ function initMap() {
 		updateMapPositionAndSize(map, bounds, zoom_offset);
 	});
 
+	map.addListener('dragend', function() {
+		saveSearchLocationWithOptionalAddress(map.getCenter());
+		location.reload();
+	});
+
+	var lastRight = null;
+	$new_search_radius=search_radius;
+	map.addListener('mouseup',function(){
+		if (event.which === 3){
+			var newRight = event.timeStamp;
+			
+			if(newRight - lastRight < 400){
+				$new_search_radius*=2;
+				setSearchRadius($new_search_radius);
+			}
+			lastRight=newRight;
+		}
+	});
+
+	map.addListener('dblclick',function(){
+		$new_search_radius/=2;
+		setSearchRadius($new_search_radius);
+	});
```

Comment:
Reword the "WIP" commit.  "wip" is mainly as a way to save changes and do something else on another branch.  It would be nice to keep them out of master.

I suggestion using "fixup" or "squash" to combine these 3 commits into one.  I think the change from all 3 together would be easier to describe:
![image](https://user-images.githubusercontent.com/11838191/43580102-32ff70ce-9622-11e8-99f5-5e0b402ceb26.png)